gammapy/maps/wcs.py

```diff
@@ -634,6 +635,40 @@ def solid_angle(self):
 
         return u.Quantity(dx * dy, 'sr')
 
+    def get_region_mask_array(self, region):
+        """Return mask of pixels inside region in the the form of boolean array
+
+        TODO: implement list of region for each axis
```

Comment:
I think in my last comment I suggested to remove this TODO (and also in the code below) and to instead show how to do this via a docs example.

Reply:
Well it is not so easy to do actually. I will add an issue about this.